README.md

```diff
@@ -87,6 +87,8 @@ Highly recommend it before preparation. After a couple of years of design you wi
 something new for you.<br>
 **Pricing**: whole course for ~ $79.
 
+### Mastering the Software Engineering Interview (University of California @ Coursera) 💰 
```

Comment:
Cool, thanks for this link! I guess that you could listen to it for free without the certificate. Could you please change it to 🆓 / 💰?